lms/templates/courseware/course_updates.html

```diff
@@ -1,11 +1,10 @@
 <%! from django.utils.translation import ugettext as _ %>
 <section aria-labelledby="course-updates-heading">
-    <h2 class="hd hd-2 sr" id="course-updates-heading">${_('All course updates')}</h2>
```

Comment:
@luckyv you have to change the tests in the same PR-- you can't merge a PR with failing tests. You can see the tests by clicking on the red X's below.

Note that the a11y failure is that this element being deleted is linked to another element on the page by ID. You will need to talk to @cptvitamin about what to do in this case.